.github/workflows/check-stable-dependency.yaml

```diff
@@ -0,0 +1,7 @@
+name: Check Stable Dependencies 
+on: [pull_request, workflow_dispatch]
+
+jobs:
+  check_dependency:
+    uses: IABTechLab/uid2-shared-actions/.github/workflows/shared-check-stable-dependency.yaml@main
```

Comment:
I know you copied these across, but we should move all the references to workflow or actions in uid2-shared-actions to versions, rather than @main. So this should be @v2.0